crates/test-utils/src/minicore.rs

```diff
@@ -822,6 +823,29 @@ pub mod ops {
     }
     pub use self::coroutine::{Coroutine, CoroutineState};
     // endregion:coroutine
+
+    // region:dispatch_from_dyn
+    mod dispatch_from_dyn {
+        use crate::marker::Unsize;
+
+        #[unstable(feature = "dispatch_from_dyn", issue = "none")]
```

Comment:
fwiw we haven't recorded stability attributes in minicore so far so no need to do that here